src/view-registry.coffee

```diff
@@ -124,6 +124,24 @@ class ViewRegistry
   # * `object` The object for which you want to retrieve a view. This can be a
   #   pane item, a pane, or the workspace itself.
   #
+  # ## View Resolution Algorithm
+  #
+  # The view associated with the object is resolved using the following
+  # heuristic sequence
+  #
+  #  1. Is the object an instance of `HTMLElement`? If true, return the object.
+  #  2. Does the object have a property named `element` with a value which is
+  #     an instance of `HTMLElement`? If true, return the property value.
+  #  3. Is the object a SpacePen view, which means it has a `jquery` property?
```

Comment:
I think this can be phrased more generally in terms of jQuery...

> 3. Is the object is a jQuery wrapper object, indicated by the presence of a `jquery` property?